app/models/miq_worker/runner.rb

```diff
@@ -487,8 +466,9 @@ def self.log_ruby_object_usage(top = 20)
   # received from the container management system (aka OpenShift).  The SIGINT
   # trap is mostly a developer convenience.
   def setup_sigterm_trap
-    Kernel.trap("TERM") { @sigterm_received = true }
-    Kernel.trap("INT")  { @sigterm_received = true }
+    INTERRUPT_SIGNALS.each do |signal|
+      Kernel.trap(signal) { @worker_should_exit = true }
```

Comment:
For clarification, this only works on non-puma workers, right?  Puma will intercept these signals before us, right?

Reply:
Yeah, I put that info in the commit message that deals with the puma exception handling and changes to the web server runner mixin.

Reply:
Doh, your code is too readable here, there was no reason to read the commit message 😆